### DISCUSSION TITLE
Brewing Guide

### DISCUSSION THREAD

**Discussion by PAPISOP** [BedrockTweaks/Files#104] [Ideas]
i was planning to add new potions ingredients which were added in 1.21 to brewing guide but it looks like even vanilla tweaks didnt update it so idk what to do lol
like should i add it or wait for vanilla tweaks update then add it?

**Comment by drav0011** [comment]
It's ok to get a ahead, just have in mind in the case their textures are different it would need to be updated again

**Reply by PAPISOP** [reply]
Alright then, today I will make it and make a pull request
